test/tools/WebListener/WebListener.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <Description>A simple ASP.NET Core 2.0 MVC app to provide an HTTP and HTTPS server for testing.</Description>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.All" Version="2.0.0" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <DotNetCliToolReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Tools" Version="2.0.0" />
+  </ItemGroup>
+
+  <ItemGroup>
+     <None Include="ServerCert.pfx" CopyToPublishDirectory="PreserveNewest" />
```

Comment:
Maybe put it near client certificate and remove the group?

Reply:
resolved